indexing-service/src/main/java/io/druid/indexing/common/task/AbstractTask.java

```diff
@@ -28,15 +28,19 @@
 import io.druid.indexing.common.actions.LockListAction;
 import io.druid.query.Query;
 import io.druid.query.QueryRunner;
-import java.util.Map;
 import org.joda.time.DateTime;
 import org.joda.time.Interval;
 
 import java.io.IOException;
+import java.util.Map;
 
 public abstract class AbstractTask implements Task
 {
   private static final Joiner ID_JOINER = Joiner.on("_");
+  protected static final int REALTIME_TASK_PRIORITY = 100;
+  protected static final int INDEX_TASK_PRIORITY = 75;
+  protected static final int MERGE_TASK_PRIORITY = 50;
+  protected static final int DEFAULT_TASK_PRIORITY = 25;
```

Comment:
hmm, IMO `null` or value-not-present for realtime should mean `100` (the default for realtime) but a literal `0` should really mean `0`.